README.md

```diff
@@ -6,46 +6,91 @@ schedgen
 
 This is a schedule image generator for streamer teams!
 
-How to use
-----------
+Software Requirements
+---------------------
+- [poetry](https://python-poetry.org): package manager (it should install summon as dev dependency)
+- Run `fc-match <FontName>` to get OS-hosted font file (change `<FontName>` for a font name of your choice)
```

Comment:
I believe fontconfig is pretty much Linux-only.

Reply:
Perhaps mac users will get this. I cannot ensure it for Windows users tho, so I'll just remove this line